db/R/get.trait.data.R

```diff
@@ -97,15 +102,15 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon,
           if (!file.exists(file.path(files$file_path[[id]], files$file_name[[id]]))) {
             foundallfiles <- FALSE
             logger.error("can not find posterior file: ", file.path(files$file_path[[id]], files$file_name[[id]]))
-          } else if (forceupdate && (files$file_name[[id]] == "species.csv")) {
+          } else if (files$file_name[[id]] == "species.csv") {
```

Comment:
I don't know what the original was trying to do. With my changes above, we'll only be in here if `!forceupdate`, i.e., we're allowing an automatic update, so there's no need to think about the value of the flag within the block.